networkx/algorithms/traversal/tests/test_reachability.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+
+import networkx as nx
+from networkx.utils import edges_equal
+
+
+class TestReachability:
```

Comment:
I have a slight preference for getting rid of the class here since it's not really necessary. I'd vote for just making the tests individual functions instead.